src/tpm2_wrap.c

```diff
@@ -2462,9 +2462,9 @@ int wolfTPM2_ReadPCR(WOLFTPM2_DEV* dev, int pcrIndex, int hashAlg, byte* digest,
     int rc;
     PCR_Read_In  pcrReadIn;
     PCR_Read_Out pcrReadOut;
-    int digestLen;
+    int digestLen = 0;
```

Comment:
Don't understand the change to init this... it is always set before use below.

Reply:
In the wolfSSL forum, Hadrien mentioned that for him, the TPM chip on his computer does not return TPM_RC_VALUE when pcrIndex is above 23. The code continues execution and the following XMEMCPY is executed with uninitialized digestLen leading to a crash. Therefore, I thought it is a good thing to initialize digestLen to zero, to eliminate potential crashes for the user.